src/discord_app.py

```diff
@@ -90,8 +90,14 @@ async def answer_question_basic(client_openai, context, question):
 
 client = discord.Client(intents=intents)
 
-client_openai = OpenAI()
```

Comment:
Similarly, having this step here complicates testing because it runs on import. I believe moving it under `answer_question_basic` works. (Doing so does mean that the client is initialized on every `answer_question_basic` call, but I do not believe it is an expensive step.)